docs/config/conf.py

```diff
@@ -22,8 +24,12 @@
 author = "Intel"
 
```

Comment:
I guess you should bump the above dates now

Reply:
done